### PR TITLE
Release v1.5.0

### PR DESCRIPTION
**Pull request by carlosalberto**

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
We need a 1.5.0 schema file before the release. I will submit a PR.

**Comment by tigrannajaryan** [comment]
Schema file added in https://github.com/open-telemetry/opentelemetry-specification/pull/1802
Need to merge before this.

**Review by tigrannajaryan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ release.
...
+
+### Semantic Conventions
+
+### Compatibility
+
+### OpenTelemetry Protocol
+
+### SDK Configuration
+
+## v1.5.0 (2021-07-07)
```
Comment (arminru) [inline CHANGELOG.md:28]: I think these PRs should also be mentioned in the changelog:
https://github.com/open-telemetry/opentelemetry-specification/pull/1738, https://github.com/open-telemetry/opentelemetry-specification/pull/1741, https://github.com/open-telemetry/opentelemetry-specification/pull/1775

Comment (carlosalberto) [reply CHANGELOG.md:28]: #1775 is more of an editorial change (the actual change happened before)

Added the other two ones, totally missed them ;)

**Comment by tigrannajaryan** [comment]
We need to merge https://github.com/open-telemetry/opentelemetry.io/pull/629 after this release is made.

**Comment by tigrannajaryan** [comment]
Please rebase to fix the build.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,7 +11,30 @@ release.
...
```
Comment (arminru) [inline CHANGELOG.md:28]:
```suggestion
## v1.5.0 (2021-07-08)
```

**Review by arminru — APPROVED** [review]

**Pull request merged**